### ISSUE TITLE
Unapproved application accessible publicly

### ISSUE THREAD

**Issue by pbek** [nextcloud/appstore#698]
I submitted an app https://apps.nextcloud.com/apps/qownnotes for the great, new #670.

I cannot see it on https://apps.nextcloud.com/account/integrations, but still https://apps.nextcloud.com/apps/qownnotes was instantly accessible for me (also in a private browser session).

https://apps.nextcloud.com/account/integrations says `You have not submitted any integration yet or they are pending approval!`.

**Comment by mario** [comment]
Thanks for the report. I do know about this and I'll fix it, but it's not critical since everyone apart from you is unlikely to know that URL.

**Comment by pbek** [comment]
Yes, I think you are right, since (I guess 😁) it's not possible to ad javascript code (that could set malicious cookies) to an app submission.
